### PR TITLE
Wrote booking class

### PR DESCRIPTION
**Pull request by justin-obrien-smoothstack**

### CODE REVIEW

**Review by trevorhuis-smoothstack — APPROVED** [review]
good entity!

File: src/main/java/com/ss/training/utopia/counter/entity/Booking.java [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+@Entity
+@Table(name = "tbl_booking")
+@IdClass(BookingPk.class)
+public class Booking implements Serializable {
+
+	private static final long serialVersionUID = -3688288514632867852L;
+
+	@Id
+	@Column
+	private Long travelerId, flightId;
```
Comment (sean-darsie) [inline src/main/java/com/ss/training/utopia/counter/entity/Booking.java:23]: Are these values longs in the database?

**Review by sean-darsie — APPROVED** [review]
Approved  with 1 comment

File: src/main/java/com/ss/training/utopia/counter/entity/Booking.java (hunk 1 above)
Comment (justin-obrien-smoothstack) [reply src/main/java/com/ss/training/utopia/counter/entity/Booking.java:23]: They're integers, which use the same number of bits as Java integers, but are unsigned so their maximum value is twice as large (as I explained during our group meeting today).

**Pull request merged**